<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tcp_Server</class>
 <widget class="QWidget" name="Tcp_Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>541</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>端口：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txt_Port"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="btn_Listen">
      <property name="text">
       <string>侦听</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QTextEdit" name="textEdit_Rev"/>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QTextEdit" name="textEdit_Send"/>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="btn_Send">
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
